python/ray/serve/_private/router.py

```diff
@@ -202,10 +204,12 @@ def update_deployment_config(
                     ),
                 )
                 # Push metrics to the controller periodically.
-                self.metrics_pusher.register_or_update_task(
-                    self.PUSH_METRICS_TO_CONTROLLER_TASK_NAME,
-                    self.push_autoscaling_metrics_to_controller,
-                    autoscaling_config.metrics_interval_s,
```

Comment:
@zcin @edoakes I'm working on resurrecting this PR and I think this was the main sticking point - the shared metrics pusher wouldn't respect the `autoscaling_config.metrics_interval_s`. Can we deprecate that parameter? Would we need to feature-flag this PR so it can be introduced gradually? Or maybe we want to preserve it and have a different shared pusher for each `metric_interval_s`?

Reply:
I am fine with making the `metrics_interval_s` a cluster-level option configured via env var. WDYT Cindy?

Reply:
Ah! I like that option!

Reply:
Yes I think making the metrics interval a cluster level option makes sense.

Reply:
Sounds good - I will take that approach when I get back from the holidays